docs/data/install-xroot-client.md

```diff
@@ -1,97 +1,228 @@
+Using XRootD 
+=============
 
-Install XRootd Client 
-=====================
+XRootD is a high performance data system widely used by several science VOs on OSG to store and to distribute data to
+jobs.
+It can be used to create a data store from distributed data nodes or to serve data to systems using a distributed
+caching architecture.
+Either mode of operation requires you to install the XRootD client software.
 
-XRootD is a high performance network storage system widely used in high energy physics experiments such as ATLAS and ALICE. The underlying XRootD data transfer protocol provides highly efficient access to ROOT based data files.
+This page provides instructions for accessing data on XRootD data systems using a variety of methods.
 
-This page provides instructions for using a XRootD client to connect to a XRootD storage system.
+As a user you have three different ways to interact with XRootD: 
 
-Installing a XRootD Client
---------------------------
+1. Using the XRootD clients
+1. Using a XRootDFS FUSE mount to access a local XRootD data store
+1. Using LD\_PRELOAD to use XRootD libraries with Unix tools
```

Comment:
Let's make `XRootD clients`, `XRootdFS FUSE mount` and `LD\_PRELOAD` links to the relevant sections

Reply:
Done